db/migrations/schema-objects/20230609145304.do.hdf-deletion-columns.sql

```diff
@@ -0,0 +1,39 @@
+IF NOT EXISTS(SELECT *
+                FROM sys.columns
+               WHERE object_ID = object_id('[mtc_admin].[hdf]')
+                 AND col_name(object_ID, column_Id) = 'isDeleted')
+  BEGIN
+  ALTER TABLE [mtc_admin].[hdf]
+      ADD [isDeleted] BIT CONSTRAINT [DEFAULT_hdf_isDeleted] DEFAULT 0 NOT NULL;
+  END
+
+IF NOT EXISTS(SELECT *
+                FROM sys.columns
+               WHERE object_ID = object_id('[mtc_admin].[hdf]')
+                 AND col_name(object_ID, column_Id) = 'deletedAt')
+  BEGIN
+  ALTER TABLE [mtc_admin].[hdf]
+      ADD [deletedAt] DATETIME2 (3) NULL;
```

Comment:
Other deletedAt cols use DATETIMEOFFSET e.g. pupilRestart

Reply:
good catch, gotta be careful with that copilot autocomplete!